### PR TITLE
WIP: Eva/layers

### PR DESCRIPTION
**Pull request by gevos**

### CODE REVIEW

**Comment by mhaertwig** [comment]
Nice! If you want me to merge, please remove the WIP status.

File: src/config/layers.js [hunk 1]
```diff
@@ -75,6 +78,7 @@ export const netzkarteLayer = new TrafimageRasterLayer({
         matrixIds: resolutions.map((r, i) => `${i}`),
       }),
     }),
+    zIndex: -10,
```
Comment (oterral) [inline src/config/layers.js]: Do we really needs this ?  why a negative value ?

**Comment by gevos** [comment]
> I'm wondering why this isn't translated: https://github.com/geops/react-spatial/blob/master/src/components/LayerTree/LayerTree.js#L194

The layers are translated now.

File: src/config/layers.js (hunk 1 above)
Comment (oterral) [reply src/config/layers.js]: Ok fine for me.

File: src/__snapshots__/AppRouter.test.js.snap [hunk 2]
```diff
@@ -166,7 +166,7 @@ exports[`AppRouter should match snapshot 1`] = `
         <div
           className="wkp-menu-layers hidden"
         >
-          Stationen, Stationen, Flughäfen, Bahnhofpläne, Printprodukte
+          ch.sbb.ch_gemeinden, ch.sbb.netzkarte.buslinien, Stationen, Stationen, Flughäfen, Bahnhofpläne, Printprodukte
```
[the remark this answers is not in the document]
Comment (oterral) [reply src/__snapshots__/AppRouter.test.js.snap]: The 't' function must be given as props of the LayerTree component.

**Pull request merged**